docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -20,6 +20,8 @@ sort_order: 4
 
 # GitHub Enterprise user licences in the BC government
 
+>Note: This page is under review as of December 2023 and will be updated for clarity and accuracy early in 2024. 
```

Comment:
if this is still here in June, please lodge a formal compliaint with the management